scripts/dev-environment.sh

```
@@ -571,29 +619,43 @@ show_status() {
         echo "   JSON-RPC: $JSON_RPC_URL"
         echo "   Logs: core-lane.log"
-        if [ $CORE_LANE_TAIL_PID -ne 0 ] && kill -0 $CORE_LANE_TAIL_PID 2>/dev/null; then
-            echo "   Log viewer: Running (PID: $CORE_LANE_TAIL_PID)"
-        fi
     else
         echo "❌ Core Lane Node: Not running"
     fi
 
-    if [ $MINING_PID -ne 0 ] && kill -0 $MINING_PID 2>/dev/null; then
-        echo "✅ Mining Loop: Running (PID: $MINING_PID)"
+    local mining_pid=$(pgrep -f "dev-environment.sh" | head -1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)